dex/pair/src/lib.rs

```diff
@@ -83,9 +87,16 @@ pub trait Pair<ContractReader>:
         self.initial_liquidity_adder()
             .set(&initial_liquidity_adder.into_option());
```

Comment:
Since it's not possible to skip this argument while also giving admins, I suggest using `ManagedAddress` for argument, and storing `Option::None` if the argument is the zero address. (since this requires the least code changes).

Additionally, make sure you change the code in the `router` contract that deploys pair contracts as well.

Reply:
Done.